assets/helpers/subscriptions.js

```diff
@@ -27,8 +27,9 @@ export type ComponentAbTest = {
 
 
 // ----- Config ----- //
-export type BillingPeriod = 'sixweek' | 'quarter' | 'year' | 'month';
+export type BillingPeriod = 'sixweek' | 'quarter' | 'year' | 'month' | 'sixday' | 'weekend' | 'sunday';
```

Comment:
we're offering more billing options?

Reply:
`BillingPeriod` might not be the best place for this construct. Basically, paper is four different subscription plans rather than three different billing periods (like GW)